js/viz/series/points/bar_point.js

```diff
@@ -45,6 +45,19 @@ export default _extend({}, symbolPoint, {
         }
     },
 
+    _calculateVisibility: function(x, y, width, height) {
+        const that = this;
+        const visibleArea = that._getVisibleArea();
```

Comment:
We can immediately extract `minX` and `minY` through destructurization  to make the code clearer
```suggestion
        const { minX, minY } = that._getVisibleArea();
```